MAINTAINERS

```diff
@@ -508,3 +510,7 @@ M: Ira Cooper
 M: Shwetha Panduranga
 M: Nithya Balachandran
 M: Raghavendra Gowdappa
+M: Susant Palai
+M: Barak Sason Rofman <bsasonro@redhat.com>
```

Comment:
we don't need to move the active participants here!, so your name, and below 2 can be removed.

Reply:
Done